### DISCUSSION TITLE
WebGL data collection errors w/ AWS

### DISCUSSION THREAD

**Discussion by Dawson-Petersen** [immersivecognition/unity-experiment-framework#69] [Q&A] [answered]
Hello all, 

I've been trying to get data collection with AWS set up for my WebGL experiment, but I'm getting some errors that I don't really understand. I set up my AWS following the instructions on [this page](https://github.com/immersivecognition/unity-experiment-framework/wiki/AWS-DynamoDB-setup). Any idea what's causing these errors? 

![Unsafe header etag](https://user-images.githubusercontent.com/77287912/113520892-08b0ba00-9564-11eb-9f54-55dcac30c592.JPG)
![AWS_Error](https://user-images.githubusercontent.com/77287912/113520893-09495080-9564-11eb-91d8-7691eb640729.JPG)

**Comment by jackbrookes** [comment, marked as answer]
Hi,
Hard to say, but looks like your credentials are not set up correctly

https://github.com/immersivecognition/unity-experiment-framework/wiki/AWS-DynamoDB-setup#putting-the-credentials-into-your-project

Make sure the credentials you created are applied to the Data Handler, and that they dont have spaces at the end, for example

**Reply by Dawson-Petersen** [reply]
It looks like I didn't include the region code in the identity pool spot. I must have misunderstood the instructions page. It's working now. Thank you!

**Reply by jackbrookes** [reply]
I'll try to add a screenshot to make it clearer! I wrote the article in a bit of a rush
